django/contrib/messages/storage/cookie.py

```diff
@@ -86,6 +86,7 @@ def _update_cookie(self, encoded_data, response):
                 domain=settings.SESSION_COOKIE_DOMAIN,
                 secure=settings.SESSION_COOKIE_SECURE or None,
                 httponly=settings.SESSION_COOKIE_HTTPONLY or None,
+                samesite=settings.SESSION_COOKIE_SAMESITE or None,
```

Comment:
Isn't this `or None` intended for the case when the `settings.SOMETHING` isn't defined at all?